hand_eye_calibration/bin/tf_to_csv.py

```diff
@@ -66,7 +66,7 @@ def write_transformation_to_csv_file(bag_file, target_frame, source_frame,
           # Write to csv file.
           quaternion = np.array(hamilton_quaternion)
           csv_file.write(
-              str(single_tf.header.stamp.to_sec()) + ', ' +
+              str("{:.9f}".format(single_tf.header.stamp.to_nsec()/1000000000.0)) + ', ' +
```

Comment:
i think the to_sec() conversion should work here. so all what you need is: 
```
str("{:.9f}".format(single_tf.header.stamp.to_sec()))
```

Reply:
yeah that works as well